porcupine/plugins/highlight/tree_sitter_highlighter.py

```diff
@@ -140,6 +140,9 @@ def _decide_tag(self, node: tree_sitter.Node) -> str:
         else:
             type_name = node.type
 
+        if set(type_name) <= set("+-*/%~&|^!?<>="):
+            return "Token.Operator"
```

Comment:
This `return` statement is before getting from the `.yml` file, so it's impossible to override this in the `.yml` file. Maybe something like this:

```suggestion
            default = "Token.Operator"
        else:
            default = "Token.Text"
```
And then use `default` instead of `"Token.Text"` below.